valhalla/midgard/util.h

```diff
@@ -36,6 +36,9 @@ template <class T> struct ranged_default_t {
   }
 };
 
+// Applied if time or distance were omitted in the request
+constexpr float kNoIsoMetric = 0.0f;
```

Comment:
prob not the best place for it.. will change for sure

Reply:
any recommendations? btw, this goes for quite a few methods/constants.. not sure if I placed them right. though here I'm sure it's not the right place;)